opdracht4.js

```diff
@@ -0,0 +1,193 @@
+//Opdracht 4 -Functies
+//Maak deze gehele opdracht eerst alsof je het voor één tv doet. We gaan één tv weergeven in het volgende format:
+//Philips 43PUS6504/12 - 4K TV
+//€379,-
+//43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+//NIKKEI NH3216SMART - HD smart TV
+//€159,-
+//32 inch (81 cm)
+//Opdracht 4a: Zorg ervoor dat er een string wordt gegenereerd voor de naam van een tv. Maak een functie die één enkel
+// tv-object (zoals inventory[0] of inventory[6]) verwacht en de naam op de volgende manier samenvoegt:
+// [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV.
+// Test of jouw functie ook werkt wanneer er een ander tv object wordt meegegeven.
+
+function tvName(tvObject){
+    const brandName = tvObject.brand;
+    const typeName = tvObject.type;
+    const tvName = tvObject.name;
```

Comment:
Je zou het object destructering kunnen toepassen:
```suggestion
    const { brand: brandName, type: typeName, name: tvName } = tvObject;
```